tests/contrib/litellm/conftest.py

```diff
@@ -2,33 +2,88 @@
 
 from ddtrace.contrib.internal.litellm.patch import patch
 from ddtrace.contrib.internal.litellm.patch import unpatch
+from ddtrace.llmobs import LLMObs as llmobs_service
+from ddtrace.llmobs._constants import AGENTLESS_BASE_URL
+from ddtrace.llmobs._writer import LLMObsSpanWriter
 from ddtrace.trace import Pin
 from tests.contrib.litellm.utils import get_request_vcr
 from tests.utils import DummyTracer
-from tests.utils import DummyWriter
+from tests.utils import override_global_config
+
+
+class TestLLMObsSpanWriter(LLMObsSpanWriter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.events = []
+
+    def enqueue(self, event):
+        self.events.append(event)
+
+
+def default_global_config():
+    return {}
+
+
+@pytest.fixture
+def ddtrace_global_config():
+    return {}
+
+
+@pytest.fixture
+def llmobs_span_writer():
+    agentless_url = "{}.{}".format(AGENTLESS_BASE_URL, "datad0g.com")
+    yield TestLLMObsSpanWriter(is_agentless=True, agentless_url=agentless_url, interval=1.0, timeout=1.0)
```

Comment:
Same - this will probably need to be modified once https://github.com/DataDog/dd-trace-py/pull/12966/ is merged